src/util/exit.js

```diff
@@ -1,8 +1,7 @@
-module.exports = code => {
+module.exports = code => new Promise(() => {
```

Comment:
Lol surprised this didn't trigger a linter error.

cc @sindresorhus - `unicorn/` plugin doesn't do this? O.o

Reply:
Oh nevermind, we don't use XO in the cli.